participation.html

```diff
@@ -15,11 +15,12 @@
+        <p>Troop 370 is a boy-led troop, meaning that the scouts lead the meetings and plan the activities and campouts - not the adults. All scouts have the opportunity to take leadership positions in the troop, such as being a Senior Patrol Leader, Assistant Senior Patrol Leader, Instructor, Troop Guide, Patrol Leader, Scribe, Quartermaster, or Historian. Scouts can take leadership positions at any time or for any purpose; they do not need to wait until elections or for when they need it for advancement. One of the primary rolesa of leaders in the troop is planning the meetings in the monthly PLCs (patrol leader councils). Any scout wanting to give their input for the weekly meetings can do so by attending the PLCs. Read more about our leadership in our <a href="src/documents/Guidelines%20for%20Positions%20of%20Leadership.pdf">Guidelines for Positions of Leadership document</a></p>
         <p>Advancement moves at the pace of the scout, so he can move as quickly or as slowly as he wants or as time permits. Troop instructors and assistant scoutmasters are happy to help any scout with the advancement requirements. </p>
         <p>Overall, there are many opportunities for fun, service, and leadership at the Troop, and scouts can participate in as many or as few things as they would like!</p>
-        <p>For parents, there are many ways in which you can assist through volunteer efforts.  There is no greater joy than joining your Scout through their Troop 370 experience, and working to support the Troop "behind the scenes".  If you would like to see how you can help, please contact the Troop, attend a meeting, or check out the "What Can I Do?" link in the password protected members section of the site.  Meetings are held most Wednesday evenings from 7:30 - 9:00PM in the St. James UMC activities building.</p>
+        <p>For parents, there are many ways in which you can assist through volunteer efforts. There is no greater joy than joining your Scout through their Troop 370 experience, and working to support the Troop "behind the scenes".  If you would like to see how you can help, please contact the Troop, attend a meeting, or check out the "What Can I Do?" link in the password protected members section of the site.  Meetings are held most Wednesday evenings from 7:30 - 9:00PM in the St. James UMC activities building.</p>
```

Comment:
```suggestion
        <p>For parents, there are many ways in which you can assist through volunteer efforts. There is no greater joy than joining your Scout through their Troop 370 experience and working to support the Troop behind the scenes.  If you would like to see how you can help, please contact the Troop, attend a meeting, or check out the "What Can I Do?" link in the password protected members section of the site.  Meetings are held most Wednesday evenings from 7:30 - 9:00PM in the St. James UMC activities building.</p>
```
Remove unnecessary quotation marks